vars/highLevelDataSetup.groovy

```diff
@@ -0,0 +1,23 @@
+#!groovy
+import uk.gov.hmcts.contino.Builder
+import uk.gov.hmcts.contino.PipelineCallbacksRunner
+import uk.gov.hmcts.contino.AppPipelineConfig
+
+def call(params) {
+  PipelineCallbacksRunner pcr = params.pipelineCallbacksRunner
+  AppPipelineConfig config = params.appPipelineConfig
+  Builder builder = params.builder
+
+  def environment = params.environment
+  def product = params.product
+
+  if (config.highLevelDataSetup) {
+    stageWithAgent("High Level Data Setup - ${environment}", product) {
+      withDefinitionImportSecrets(product, environment, config.vaultEnvironmentOverrides){
```

Comment:
should we also be adding idam url, client id and redirect url as part of either secrets or `withEnv`?

Reply:
We added env vars for IDAM_URL, S2S_URLS2S_URL, CCD_API_GATEWAY_S2S_ID, CCD_API_GATEWAY_OAUTH2_CLIENT_ID and CCD_API_GATEWAY_OAUTH2_REDIRECT_URL

fixed in [commit 24826c18d969b3ac35100fa7b2b9607edb49aa86](https://github.com/hmcts/cnp-jenkins-library/pull/747/commits/24826c18d969b3ac35100fa7b2b9607edb49aa86)

Reply:
Added one more env var env.DEFINITION_STORE_URL_BASE and switched S2S_URL, IDAM_URL with S2S_URL_BASE and IDAM_URL_BASE